css/product-payment.css

```
@@ -0,0 +1,555 @@
+  text-decoration: none;
+}
+
+.product-page__add{
+  display: flex;
+  margin-top: 20px;
+  gap: 28px;
+  background-color: var(--color-whiteX);
+  padding: 19px 0px;
+  border-radius: 0px 0px 16px 16px;
+  position: relative;
+}  
+
+.product-page__quantity-selector{
+  display: flex;
+  align-items: center;
+  border: 1px solid var(--color-solid);
+  width: 115px;
+  border-radius: var(--eight-pixel);
+  justify-content: center;
+  gap: 5px;
+  position: absolute;
+  margin-left: 20px;
+  background-color: var(--color-white);
+}
+
+.product-page__quantity__link {
+  font-size: 0;
+  inset: 0;
+  position: absolute;
+  z-index: 1;
+}
+
+.product-page__decrease{
+  border: none;
+  background: none;
+  cursor: pointer;
+  width: 22px;
+  border: 1px solid var(--color-solid);
+  margin-left: 5px;
+}
+
+.product-page__quantity {
+  font-size: var(--twenty-pixel);
+  font-weight: var(--font-weight);
+  line-height: 25px;
+  text-align: center;
+  width: 95px;
+  padding: 7px 10px 10px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-right: 5px;
+  border-left: 1.5px solid var(--color-solid);
+  border-right: 1.5px solid var(--color-solid);
+  color: var(--color-solid);
+  background-color: var(--color-whiteX);
+}
+
+.product-page__increase{
+  border: none;
+  background: none;
+  cursor: pointer;
+  width: 25px;
+  border: 1.5px solid var(--color-solid);
+  margin-right: 6px;
+}
+
+.product-page__increase--1{
+  height: var(--eighteen-pixel);  
+  width: var(--zero);
+  position: absolute;
+  margin-left: 93px;
+}
+
+.product-page__add-purchases{
+  display: flex;
+  gap: var(--eight-pixel);
+  background-color: var(--color-pink);
+  border: none;
+  border-radius: var(--eight-pixel);
+  cursor: pointer;
+  box-shadow: 0px 4px 12px 0px #D14D411A;
+  padding: 10px 20px;
+  position: relative;
+  margin-top: -5px;
+  margin-right: 20px;
+}
+
+.product-page__add-purchases:hover{
+  background-color: #9e255b;
+}
+
+.product-page__add-cart{
+  font-size: var(--font-size-smalls);
+  font-weight: var(--font-weight-small);
+  line-height: var(--eighteen-pixel);
+  text-align: center;
+  color: var(--color-white);
+  letter-spacing: 0.06em;
+}
+
+.product-page__payment-methods {
+  display: flex;
+  height: 9px;
+  gap: 5px;
+  margin-top: 10px;
+}
+
+.product-page__add-target{
+  margin-left: 150px;
+}
+
+/* Desktop style */
+@media (min-width: 768px) {
+  .product-page__top {
+    padding-bottom: 24px;
+    margin: 0 5px; 
+  }
+
+  .product-page__text {
+    font-size: var(--font-size-smalls);
+  }
+  
+  .product-page__content{
+    flex-direction: row;
+    margin: 0px 0px 20px;
+    gap: 52px;
+  }
+
+  .product-page__air {
+    margin-top: 21pc;
+  }
+  
+  .product-page__image {
+    width: 150px;
+  }
+
+  .product-page__zoom {
+    width: 640px;
+  }
+
+  .product-page__img {
+    align-items: center;
+    margin: auto;
+    width: 430px;
+  }
+
+  .product-page__card{
+    width: 437px;
```

Para que width si usas flex? Vi que lo haces en las cosas que usas flex
```suggestion
    flex-basis: 438px;
```